### PR TITLE
Added two more enums buttons in kernel!

### PR DESCRIPTION
**Pull request by DavisDev**

### CODE REVIEW

File: include/psp2kern/ctrl.h [hunk 1]
```diff
@@ -38,9 +38,11 @@ enum SceCtrlPadButtons {
 	SCE_CTRL_CIRCLE      = 0x002000,	//!< Circle button.
 	SCE_CTRL_CROSS       = 0x004000,	//!< Cross button.
 	SCE_CTRL_SQUARE      = 0x008000,	//!< Square button.
+	SCE_CTRL_HP          = 0x080000,	//!< Headphone pressent.
```
Comment (devnoname120) [inline include/psp2kern/ctrl.h]: `plugged in`

File: include/psp2kern/ctrl.h [hunk 2]
```diff
@@ -38,7 +38,7 @@ enum SceCtrlPadButtons {
 	SCE_CTRL_CIRCLE      = 0x002000,	//!< Circle button.
 	SCE_CTRL_CROSS       = 0x004000,	//!< Cross button.
 	SCE_CTRL_SQUARE      = 0x008000,	//!< Square button.
-	SCE_CTRL_HP          = 0x080000,	//!< Headphone pressent.
+	SCE_CTRL_HP          = 0x080000,	//!< Headphone plugged in.
```
Comment (devnoname120) [inline include/psp2kern/ctrl.h]: Is `SCE_CTRL_HP` an official name? If it's not I think it's more clear to put `SCE_CTRL_HEADPHONE`.

File: include/psp2kern/ctrl.h [hunk 3]
```diff
@@ -38,9 +38,11 @@ enum SceCtrlPadButtons {
 	SCE_CTRL_CIRCLE      = 0x002000,	//!< Circle button.
 	SCE_CTRL_CROSS       = 0x004000,	//!< Cross button.
 	SCE_CTRL_SQUARE      = 0x008000,	//!< Square button.
+	SCE_CTRL_HP          = 0x080000,	//!< Headphone pressent.
 	SCE_CTRL_INTERCEPTED = 0x010000,        //!< Input not available because intercepted by another application
 	SCE_CTRL_VOLUP       = 0x100000,	//!< Volume up button.
-	SCE_CTRL_VOLDOWN     = 0x200000		//!< Volume down button.
+	SCE_CTRL_VOLDOWN     = 0x200000,	//!< Volume down button.
+	SCE_CTRL_PWR         = 0x40000000	//!< Power button.
```
Comment (devnoname120) [inline include/psp2kern/ctrl.h]: I suggest `SCE_CTRL_POWER` to be consistent with the other buttons of the enum.

**Comment by devnoname120** [comment]
This is actually a duplicate of #194 which I merged.

**Comment by DavisDev** [comment]
But, it's for the kernel version, is not it required to update on both?

**Comment by devnoname120** [comment]
You are right, I mixed things up.

**Pull request merged**